easybuild/easyconfigs/h/HPCX/HPCX-2.3.0-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'Bundle'
+
+name = 'HPCX'
+version = '2.3.0'
+
+homepage = 'http://www.mellanox.com/page/products_dyn?product_family=189&mtag=hpc-x'
+description = """The Mellanox HPC-X Toolkit
+is a comprehensive MPI and SHMEM/PGAS software suite
+for high performance computing environments"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+dependencies = [
+    ('UCX', '1.5.0rc1', '-hpcx', ('GCCcore', '8.2.0')),
```

Comment:
You only need
('UCX', '1.5.0rc1', '-hpcx'),
('OpenMPI', '4.0.x', '-hpcx'),

It will find the correct UCX, OpenMPI version by itself.

Reply:
Removed.